src/features/authentication/components/LoginForm/Categories/Categories.css

```diff
@@ -0,0 +1,46 @@
+.sport-card {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  border-radius: 30px; 
+  overflow: hidden;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+}
+
+.sport-image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover; 
+  border-radius: 25px; 
+}
+
+.sport-overlay {
+  position: absolute;
+
+  bottom: 1px; 
+  left: 50%;
+  transform: translateX(-50%);
+  background: rgba(177, 179, 177, 0.71); 
+  color: white;
+  text-align:start;
+  padding: 10px;
+  border-radius: 15px; 
+  font-size: 18px;
+  font-family: Arial, sans-serif;
```

Comment:
font family should also be from css variables. We can do this later, for one of the team members did this already